rdt-open-reader/src/main/java/io/ona/rdt/util/Utils.java

```diff
@@ -158,4 +162,13 @@ public static boolean tableExists(SQLiteDatabase db, String tableName) {
     public static boolean isEmptyCursor(Cursor cursor) {
         return cursor == null || cursor.getCount() == 0;
     }
+
+    public static AllSharedPreferences getSharedPreference(Context context) {
+        return new AllSharedPreferences(PreferenceManager.getDefaultSharedPreferences(context)) {
+            @Override
+            public String fetchLanguagePreference() {
+                return getPreferences().getString("locale", BuildConfig.LOCALE).trim();
```

Comment:
`RDTApplication.getInstance().getContext().allSharedPreferences().getPreference(KEY);` would work for getting a saved preference.

Reply:
done.